docs/code_navigation/how-to/index.mdx

```diff
@@ -15,8 +15,6 @@
 
 - [Add precise indexing to many repositories](/code_navigation/how-to/adding_lsif_to_many_repos)
 - [Add precise indexing to CI workflows](/code_navigation/how-to/adding_lsif_to_workflows)
-- [Enable auto-indexing](/code_navigation/how-to/enable_auto_indexing)
-- [Configure auto-indexing](/code_navigation/how-to/configure_auto_indexing)
```

Comment:
Is this `index.mdx` going to be kept around? Why are these two links being removed from it?